### PR TITLE
Update to reflect the new admin UI

### LINKED ISSUE #1499: Managing Policies chapter
### Description

Update this Authorization Services Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1499

### CODE REVIEW

File: authorization_services/topics/policy-overview.adoc [hunk 1]
```diff
@@ -12,6 +12,12 @@ image:{project_images}/policy/view.png[alt="Policies"]
 +
 On this tab, you can view the list of previously created policies as well as create and edit a policy.
 
+ifeval::[{project_community}==true]
+. To create a new policy, click *Create policy* button, then select a policy type from the list.
```
Comment (andymunro) [inline authorization_services/topics/policy-overview.adoc]:
```suggestion
. To create a new policy, click *Create policy*, then select a policy type from the list.
```

File: authorization_services/topics/policy-time-policy.adoc [hunk 2]
```diff
@@ -19,6 +19,15 @@ can identify them more easily.
 +
 A string containing details about this policy.
 +
+ifeval::[{project_community}==true]
+* *Start time*
++
+Defines the time before which access must *not* be granted. Permission is granted only if the current date/time is later than or equal to this value.
+* *Expire time *
```
Comment (andymunro) [inline authorization_services/topics/policy-time-policy.adoc]:
```suggestion
* *Expire time*
```

File: authorization_services/topics/policy-time-policy.adoc [hunk 3]
```diff
@@ -27,6 +36,11 @@ Defines the time before which access must *not* be granted. Permission is grante
 * *Not On or After*
 +
 Defines the time after which access must *not* be granted. Permission is granted only if the current date/time is earlier than or equal to this value.
+endif::[]
+
+ifeval::[{project_community}==true]
+Select *Repeat* to repeat access being granted on a specific *Day of Month*, *Month*, *Year*, *Hour* or *Minute*.
+endif::[]
 +
```
Comment (andymunro) [inline authorization_services/topics/policy-time-policy.adoc]:
```suggestion

```

Comment (andymunro) [reply authorization_services/topics/policy-time-policy.adoc]: Also, remove + (plus sign) on lines 48, 52, 56, and 64.

File: authorization_services/topics/policy-client-policy.adoc [hunk 4]
```diff
@@ -3,7 +3,12 @@
...
 
+ifeval::[{project_community}==true]
+To create a new client-based policy, select *Client* from the policy type list.
+endif::[]
+ifeval::[{project_product}==true]
 To create a new client-based policy, select *Client* in the item list in the upper right corner of the policy listing.
+endif::[]
 
 .Add a Client Policy
 image:images/policy/create-client.png[alt="Add a Client Policy"]
```
Comment (andymunro) [inline authorization_services/topics/policy-client-policy.adoc]: This is the old image.

File: authorization_services/topics/policy-client-policy.adoc [hunk 5]
```diff
@@ -21,7 +26,7 @@ A string containing details about this policy.
 +
 * *Clients*
 +
-Specifies which clients are given access by this policy.
+Specifies which clients are givenGroup-based policy access by this policy.
```
Comment (andymunro) [inline authorization_services/topics/policy-client-policy.adoc]:
```suggestion
Specifies which clients have givenGroup-based policy access by this policy.
```

File: authorization_services/topics/policy-client-policy.adoc (hunk 4 above)
Comment (andymunro) [reply authorization_services/topics/policy-client-policy.adoc]: Also, there is an old image in policy-client-scope-policy-required-client-scope.adoc

**Review by andymunro — CHANGES_REQUESTED** [review]
A few corrections, @edewit

File: authorization_services/topics/policy-time-policy.adoc [hunk 6]
```diff
@@ -27,27 +36,26 @@ Defines the time before which access must *not* be granted. Permission is grante
 * *Not On or After*
 +
 Defines the time after which access must *not* be granted. Permission is granted only if the current date/time is earlier than or equal to this value.
-+
+endif::[]
+
+ifeval::[{project_community}==true]
+Select *Repeat* to repeat access being granted on a specific *Day of Month*, *Month*, *Year*, *Hour* or *Minute*.
+endif::[]
 * *Day of Month*
```
Comment (andymunro) [inline authorization_services/topics/policy-time-policy.adoc:45]:
```suggestion

* *Day of Month*
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looking good @edewit. Just one minor change needed to fix a formatting problem.

**Review by andymunro — APPROVED** [review]
Okay to merge, @ssilvert 
Thanks, @edewit

**Review by ssilvert — APPROVED** [review]

**Pull request merged**